transport/src/main/java/io/netty5/channel/embedded/EmbeddedEventLoop.java

```diff
@@ -49,6 +49,9 @@ final class EmbeddedEventLoop extends AbstractScheduledEventExecutor implements
 
     private final Queue<Runnable> tasks = new ArrayDeque<>(2);
```

Comment:
Lets do this change in a seperate PR And also add a test if possible

Reply:
Done